openedx_proversity_reports/reports/learning_tracker_report.py

```diff
@@ -154,6 +155,22 @@ def _get_time_bewteen_sessions(self, user):
         except ValueError:
             return 0
 
+    def _get_total_sessions(self, user):
+        """
+        Calculate learner metrics for "Total of sessions".
+        Args:
+            user: User Model.
+        Returns:
+            int (Total of sessions).
+        """
+        user_profile = get_user_profile().objects.get(user_id=user.id)
+
+        try:
+            meta = json.loads(user_profile.meta)
+            return int(meta.get('session_number', 0))
```

Comment:
You can merge this line with the previous one since meta is only used here. @diegomillan

Reply:
@Squirrel18 Yes, but I prefer this for readability.

Reply:
I agree, no problem. @diegomillan